tokio-util/src/io/sync_bridge.rs

```diff
@@ -0,0 +1,82 @@
+use pin_project_lite::pin_project;
+use std::io::{Read, Write};
+use std::pin::Pin;
+use tokio::io::{AsyncRead, AsyncReadExt, AsyncWrite, AsyncWriteExt};
+
+pin_project! {
+    /// Use a [`tokio::io::AsyncRead`] synchronously as a [`std::io::Read`] or
+    /// a [`tokio::io::AsyncWrite`] as a [`std::io::Write`].
+    #[derive(Debug)]
+    pub struct SyncIoBridge<T> {
+        src: Pin<Box<T>>,
+        #[pin]
+        rt: tokio::runtime::Handle,
+    }
+}
```

Comment:
You can remove `pin_project!` here. The `Handle` type does not need to be pinned.